### ISSUE TITLE
Override inside NodeTypes folder don't get applied

### ISSUE THREAD

**Issue by jonnitto** [neos/neos-development-collection#3449]
Assume I've the following node type configuration in `NodeTypes.Override.Seo`:

```yaml
'Neos.Seo:TitleTagMixin':
  properties:
    titleOverride:
      validation: ~
```

If I put this file into the `Configuration` folder, the override takes place, if I put it into the new `NodeTypes` folder, the override doesn't gets applied…

**Comment by bwaidelich** [comment]
@jonnitto I can't look into this one currently, but I can imagine that it is related to the loading order (see https://github.com/neos/neos-development-collection/issues/3000#issuecomment-646599120)

**Comment by markuspfeifenberger** [comment]
Should be fixed by #3508

**Comment by bwaidelich** [comment]
Fixed with #3508
